src/encoded/static/components/matrix.js

```diff
@@ -195,32 +193,28 @@ class Matrix extends React.Component {
                 <div>
                     <div className="panel data-display main-panel">
                         <div className="row matrix__facet--horizontal">
-                            <div className="col-sm-5 col-md-4 col-lg-3 sm-no-padding" style={{ paddingRight: 0 }}>
+                            <div className="matrix-header" style={{ paddingRight: 0 }}>
```

Comment:
Out of curiosity, could the inline style be merged into the `matrix-header` class, or is this the easier way to go?